### PR TITLE
adding the ga tag in the header on production env. fixes #337

### LINKED ISSUE #337: Lackland AFB Locator update
Lackland locator, along with FORT SAM HOUSTON and RANDOLPH AFB needs updated as below 

## Checklist

I have…

- [x] described in detail below both the behavior I expected and the behavior I observed.
- [ ] thoroughly outlined below the steps to reproduce this issue, including relevant technical details (e.g. device, operating system, browser, browser version).
- [x] attached screenshots illustrating relevant behavior.

## Expected Behavior - Change the Physical Location of Lackland to:

1561 Stewart Street
Bldg. 1516
JBSA Lackland AFB, TX 78236

Mailing Address: 502 LRS/LGRDF, 1865 Selfridge Avenue, JBSA Lackland AFB, San Antonio, TX 78236

Please add the following:

Hours:
-Mon-Tues-Thurs-Fri 0700-1600
-Closed Wed 1400 for AETC mandatory training, Weekends, Holidays, & AETC Family Days

Notes:
-PPM Mass Briefings during peak season (June-Aug) by appt. only -Mon-Wed-Fri @ 1000 Student Mass Briefings 

Services:
- Walk-In Help
- Briefings
- Appointments

Also for location:  FORT SAM HOUSTON
Physical Location: 2484 Stanley Rd, Bldg. 2263, JBSA Ft Sam Houston
San Antonio, TX 78234

FORT SAM HOUSTON Mailing Address:  502 LRS/LGRDF, 1865 Selfridge Avenue, JBSA Lackland AFB, San 
Antonio, TX 78236

Also for location: RANDOLPH AFB
Physical Location: 555 D St East, Bldg. 399, JBSA Randolph AFB
Universal City, TX 78148

RANDOLPH AFB Mailing Address:  502 LRS/LGRDF, 1865 Selfridge Avenue, JBSA Lackland AFB, San 
Antonio, TX 78236

_Summarize expected behavior._  Please add to the Lackland AFB locator ...attached example of how the Fort Sam Houston is right below it so it looks like all being asked for is available to be entered 

## Observed Behavior - Lackland AFB missing information - See attached as it shows the area missing compared to the next one down, Fort Sam Houston 

_Summarize observed behavior._

## Steps to Reproduce

To reproduce this issue…

1. _ Enter "Lackland" into the locator and it should be the first thing that populates ...Also need to update FORT SAM HOUSTON and RANDOLPH AFB with physical and mailing addresses. 

1. _Describe step two._
1. _etc. etc. etc._

## Screenshots

_Attach relevant screenshots here._
[Lackland Locator update  GitHub #337.pdf](https://github.com/deptofdefense/move.mil/files/1661176/Lackland.Locator.update.GitHub.337.pdf)

### PR DESCRIPTION
**Pull request by rich-allen-gov**
- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [ ] thoroughly outlined below the steps necessary to test my changes.
- [ ] attached screenshots illustrating relevant behavior before and after my changes.
- [x] read, understand, and agree to the terms described in [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md).
- [x] added my name, email address, and copyright date to [CONTRIBUTORS.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTORS.md).

## Summary of Changes

Adds a second analitics tag so that non-government access to data is easier, and more filterable than with DAP

## Testing

To verify the changes proposed in this pull request…

Check your browsers network console and verify that the GA javascript files have been added.

The tag only gets put on production so I'm not sure of the testing scenario. We could add additional dev/staging accounts and use an env variable to configure if we feel that sort of testing would add value.

### CODE REVIEW

**Comment by rich-allen-gov** [comment]
Note: I messed up branch naming here, the branch references 337 however this actually fixes #383 377.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -18,6 +18,20 @@
...
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-115877489-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-115877489-1');
+    </script>
+  <%- end -%>
```
Comment (jgarber623-gov) [inline app/views/layouts/application.html.erb]: Recommend moving this code to the bottom of the page (around lines 166-169) with the existing DAP code.

**Comment by rich-allen-gov** [comment]
@jgarber623-gov updated, my thought was google was specific to have it in the head, I thought that may have to do with some of the more advanced data they collect and load order. I moved it back to the bottom with the dap script and it doesn't seem like there are any noticeable changes in how it loads so from an code organization perspective this makes sense.

**Comment by jgarber623-gov** [comment]
@rich-allen-gov Actually, one additional thing to do.

The site uses a [Content Security Policy](https://developers.google.com/web/fundamentals/security/csp/) (configured on [line 339 in `.ebextensions/nginx.conf`](https://github.com/deptofdefense/move.mil/blob/master/.ebextensions/nginx.config#L339)). Can you add `https://www.googletagmanager.com` to the `script-src` directive?

If there are additional domains that GTM calls out to from that script, they'll likely need to be added to the CSP, too.

**Comment by rich-allen-gov** [comment]
updated, I actually have a question on this now, would it make sense for me to remove this script out of that loop and something more like 

```
<%- if Rails.env.ga_key? -%>
``` 
and then use the env? I can update that pretty quickly if you think it's worth the effort and won't be trouble to coordinate.

**Comment by rich-allen-gov** [comment]
I would add the appropriate bin/setup modifications as well

**Comment by jgarber623-gov** [comment]
@rich-allen-gov 

Re: [your comment](https://github.com/deptofdefense/move.mil/pull/383#issuecomment-375003144)

I don't think we need to go the environment variable route for a few reasons:

1. The code is already wrapped in a "is this production?" conditional, so GA/GTM won't load in any other environment.
2. We're not currently collecting metrics on any environment _other than_ production. If we were, then using an environment variable like you suggest would be A Good Idea.
3. There may be a time that we pursue Number Two, but we're not doing that now, so I think it'd be best to keep things simple for the time being.

Yours is a good suggestion though and worth keeping in mind in the future.

**Comment by rich-allen-gov** [comment]
ok, that makes sense. Are there any special steps to flag the nginx change for testing in stage? If not this is good to go, I think.

**Comment by jgarber623-gov** [comment]
@rich-allen-gov Nope, nothing extra to do. Elastic Beanstalk updates the Nginx config during the deploy process based on the configuration files in the repo.

I'll rebase/merge this now and it'll head on out to staging.

**Pull request merged**